.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: "\U0001F41BBug report"
+about: "If something isn't working as expected \U0001F914."
+
+---
+
+| Question    | Answer
+| ------------| ---------------
+| Java Version | x.y.z
+| Operating System | e.g. Ubuntu/Windows/MacOS
+| Related issue | https://github.com/runelite/runelite/issues/XXX
+
+
+<!--
+- Replace this comment with your issue description.
+
+- Please complete the above table with a correct information.
+
+- Please include steps to reproduce your issue.
+
+- For general support, please use our Discord server: https://discord.gg/mePCs8U
+-->
+
+<!-- Please past your full stack trace below, if applicable -->
```

Comment:
It would be good to have instructions on finding client logs here

Reply:
I haven't worked with Runelite before, so could you inform me on those instructions?